Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:stable-alpine
+FROM nginx:mainline-alpine
```

Comment:
nginx mainline is a good idea but I think we should perform more checks on compiled dav module. I chose nginx stable because dav module failed sometimes using mainline.  Maybe add a new job in test workflow to download from a WebDAV mount point.